app/scripts/modules/core/pipeline/config/triggers/docker/dockerTrigger.html

```diff
@@ -3,11 +3,11 @@
     <div class="col-md-3 sm-label-right">
       Registry
     </div>
-    <div class="col-md-5">
-      <ui-select ng-model="trigger.registry" class="form-control input-sm">
```

Comment:
rationale here is that the registry URL might change and it doesn't make sense to break all the pipelines because the URL changes. ( Change could be due to versioning, http vs https, domain name differences etc )